app/(main)/league/[leagueId]/entry/[entryId]/week/WeekTeams.interface.ts

```diff
@@ -27,9 +27,10 @@ export interface ISchedule {
 export interface IWeekTeamsProps {
   field: ControllerRenderProps<FieldValues, string>;
   schedule: ISchedule[];
+  selectedTeams: INFLTeam[];
   userPick: string;
   // eslint-disable-next-line no-unused-vars
-  onWeeklyPickChange: (data: ChangeEvent<HTMLInputElement>) => Promise<void>;
+  onWeeklyPickChange: (teamSelect: string) => Promise<void>;
```

Comment:
This was also changed in `IWeekData`, how did you determine it is no longer a `ChangeEvent<HTMLInputElement>`?

Reply:
nit: sort these props A --> Z

Reply:
Same here. Can we use the NFL Team enum value?